cmd/placement/cmd/placement.go

```diff
@@ -122,7 +130,24 @@ func Run(ctx context.Context, options *placementoptions.Options) error {
 		resyncPeriod,
 	)
 
-	// create the kcp-scheduling-placement-controller
+	// create edgeSharedInformerFactory
+	providerConfig, _ := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+		clientcmd.NewDefaultClientConfigLoadingRules(),
+		&clientcmd.ConfigOverrides{},
+	).ClientConfig()
+	edgeViewConfig, err := configForViewOfExport(ctx, providerConfig, "edge.kcp.io")
+	if err != nil {
+		logger.Error(err, "failed to create config for view of edge exports")
+		return err
+	}
+	edgeClusterClientset, err := edgeclientset.NewForConfig(edgeViewConfig)
+	if err != nil {
+		logger.Error(err, "failed to create clientset for view of edge exports")
+		return err
+	}
+	edgeSharedInformerFactory := edgeinformers.NewSharedInformerFactoryWithOptions(edgeClusterClientset, resyncPeriod)
```

Comment:
What good do the resyncs do?

Reply:
This question is so big and fundamental that I have to cite @sttts 's famous book to answer:
<img width="892" alt="image" src="https://user-images.githubusercontent.com/8633434/218853001-31c4b9f7-c057-4b85-b3c5-e55632fb275b.png">

Reply:
Wow, that's outdated.  It does not recognize that informers recover from comm gaps by notifying about everything that was missed as soon as comm is restored.

We can address this separately, it is not just part of adding EdgePlacement informers.

Reply:
Also, resync period is usually a global setting, it's a common practice to control re-enqueing of events (e.g. for error conditions)

Reply:
I somehow have the memory that resync is not related to connection between the informer and the apiserver, in the sense that resync _can_ happen and _should_ happen even indeed disconnected to apiserver. I refreshed my memory by revisiting the implementation of the DeltaFIFO and confirmed my memory. So, IMO, the fundamentals described in the book is still valid.

But! If my understanding is wrong, please do share more pointers with me, I am sincerely more than happy to learn more on the internals. Appreciated!

With these been said, I really don't have any strong opinion on what to do with the `resyncPeriod` in edge-mc.

So which way should we go?

Reply:
This is one of the style issues that we should have a general conversation on.

"resync" is not the same as "relist", but that cited doc predates the distinction.

"resync" is distinct from the requeuing to retry.

"resync" redoes notifications for objects that exist, not for objects that are absent.  So its ability to cover for bugs is limited.